### DISCUSSION TITLE
Freeze at about 1,5 GB mem usage

### DISCUSSION THREAD

**Discussion by wofo99** [fedorovvl/tso_client#372] [General]
sorry for being a pain in the neck: does anyone else observe this? Almost all my guild members experience this ..

**Comment by skelgaard** [comment]
client.exe --x64

**Reply by wofo99** [reply]
thx, did solve it. I was not aware of this switch..

**Reply by Bob-282828** [reply]
can I please ask how you apply "--x64" ?

I'm having this 1.5GB mem problem too

**Reply by fedorovvl** [reply]
latest build has new settings window where u can enable x64 version without using arguments.. 
![изображение](https://github.com/fedorovvl/tso_client/assets/31754742/1bfc607d-0361-4930-8eed-548131238c21)

**Reply by Bob-282828** [reply]
Awesome  -  many thanks !

**Reply by wofo99** [reply]
hi. Being an old-fashioned guy I used a environment variable to set all switches, then start the client with it as parameter:
start .\Client.exe %ClientOptions%
Now just added the --x64 option to it..

**Comment by skelgaard** [comment]
don't see a rebuild in actions ?

**Comment by bellejt** [comment]
I am on dutch server same version 1.5.7.0 but do not get that button

**Reply by fedorovvl** [reply]
download latest build
![изображение](https://github.com/fedorovvl/tso_client/assets/31754742/c4df455e-bbea-4d0e-8b87-504f035e3244)

**Comment by bellejt** [comment]
done but gives error when choosing dutch.Will stay on version without option.THX

**Reply by fedorovvl** [reply]
what error?

**Comment by bellejt** [comment]
when i choose dutch and V x64 
![afbeelding](https://github.com/fedorovvl/tso_client/assets/118830451/277657ac-8439-4e99-887f-7218cd54a8a5)

**Comment by iraxxxx** [comment]
I have the same problem on PL server
